src/components/Nav/Mobile.tsx

```diff
@@ -285,11 +286,15 @@ const MobileNavMenu: React.FC<IProps> = ({
       />
       <IconButton
         icon={
-          <GlyphButton viewBox="0 0 24 40">
+          <GlyphButton 
+          viewBox="0 0 24 40"
+          pointerEvents={isSearchOpen||isSearchOpen&&isMenuOpen ? "none" : "auto"}
```

Comment:
```suggestion
          pointerEvents={ isSearchOpen && isMenuOpen ? "none" : "auto"}
```

The OR check isn't needed here